package.json

```
@@ -3,11 +3,10 @@
   "version": "1.0.0",
   "dependencies": {
     "chai": "4.2.0",
-    "fecha": "https://github.com/taylorhakes/fecha/archive/91680e4db1415fea33eac878cfd889c80a7b55c7.tar.gz",
-    "test-npm-dep": "https://elinah@bitbucket.org/elinah/test-npm-dep.git",
-    "cachito-npm-without-deps": "github:cachito-testing/cachito-npm-without-deps.git"
+    "is-positive": "https://github.com/kevva/is-positive.git",
```

Maybe the Guthub URL wasn't clear, can you make this change please?

```suggestion
    "is-positive": "github:kevva/is-positive.git",
```

---

done